package.json

```diff
@@ -66,6 +66,7 @@
     "remarkable": "^2.0.1",
     "reselect": "^4.1.5",
     "url": "^0.11.0",
+    "values.js": "^2.1.1",
```

Comment:
@AbhiPatel10, Can you describe why we need this package?

Reply:
This package generates colors.

When I give one color to this package, this will generate colors from 0 to 100% transparency.

Reply:
Do we really need a package for achieving that? can't we write a javascript function for generation colors from 0 to 100% transparency for given color?

Reply:
Okay let me try

Reply:
@Sachin-chaurasiya bhai, I have removed value.js package and write my own JavaScript code please have a look and review my play.